code/API_definitions/qod-api.yaml

```diff
@@ -329,18 +414,213 @@ components:
+          description: |
+            The status of the QoS profile.  The status is set to ACTIVE when the profile is created.
+            The status is set to INACTIVE when the profile is deleted.
+        targetMinUpstreamRate:
+          $ref: "#/components/schemas/Rate"
+          description: |
+            This is the target minimum upstream rate for the QoS profile.
+
+            For 5G networks 3GPP Guaranteed Bit Rate (GBR) refers to a dedicated, fixed data rate assigned to
+            specific services, ensuring a minimum performance level. As per 3GPP TS 23.203,
+            GBR is a QoS parameter used to manage traffic classes in mobile networks. It
+            provides a stable data rate for latency-sensitive applications, such as voice calls or
+            video streaming, helping maintain a consistent user experience.
+
+            The committed data rate allocated to specific services, ensuring a consistent level of
+            performance. DOCSIS networks, the "Minimum Reserved Traffic Rate" is defined in the 
+            DOCSIS 3.1 - MAC and Upper Layer Protocols Interface Specification" 
+            and it ensures a consistent level of performance for specific services within the network.
+        maxUpstreamRate:
+          $ref: "#/components/schemas/Rate"
+          description: |
+            The maximum best effort data
+            Equal to targetMinimumUpstreamRate when max is the as the target. 
+        maxUpstreamBurstRate:
+          $ref: "#/components/schemas/Rate"
+          description: |
+            The burst rate for best effort data
+        targetMinDownstreamRate:
+          $ref: "#/components/schemas/Rate"
+          description: |
+            This is the target minimum downstream rate for the QoS profile.
+
+            For 5G networks 3GPP Guaranteed Bit Rate (GBR) refers to a dedicated, fixed data rate assigned to
+            specific services, ensuring a minimum performance level. As per 3GPP TS 23.203,
+            GBR is a QoS parameter used to manage traffic classes in mobile networks. It
+            provides a stable data rate for latency-sensitive applications, such as voice calls or
+            video streaming, helping maintain a consistent user experience.
+
+            The committed data rate allocated to specific services, ensuring a consistent level of
+            performance. DOCSIS networks, the "Minimum Reserved Traffic Rate" is defined in the 
+            DOCSIS 3.1 - MAC and Upper Layer Protocols Interface Specification" 
+            and it ensures a consistent level of performance for specific services within the network.
+        maxDownstreamRate:
+          $ref: "#/components/schemas/Rate"
+          description: |
+            The maximum best effort rate
+            Equal to targetMinimumUpstreamRate when max is the as the target. 
+        maxDownstreamBurstRate:
+          $ref: "#/components/schemas/Rate"
+          description: |
```

Comment:
`description: The maximum amount of data that can be sent during any interval equal to the Packet Delay Budget, above which the latency for the packets in that burst may exceed the specified Packet Delay Budget`